src/select/interfaces.ts

```diff
@@ -142,6 +142,13 @@ export interface BaseSelectProps
    * Called when input focus is set onto the UI control.
    */
   onFocus?: NonCancelableEventHandler;
+
+  /**
+   * Specifies if the control is readOnly, which prevents the
+   * user from modifying the value and opening the dropdown, but does not prevent the value from
+   * being included in a form submission. A readOnly control is still focusable.
```

Comment:
"but does not prevent the value from being included in a form submission" - This seems to be a semi generic description for all the docs but components like Select currently does not submit any values if they are in a form. With this description could it not give the impression that it would be included?

Sandbox example: https://codesandbox.io/p/sandbox/sad-wildflower-dwd9n3

Reply:
yes, agree, this behavior applies to native form controls like `input` type `checkbox`, `radio` etc. removed this statement